website/blog/2021-04-08-GAAD-March-Accessibility-Issue-Update.md

```diff
@@ -0,0 +1,75 @@
+
+While this work continues, the React Native and Accessibility teams at Facebook are also busy continuing to evaluate accessibility bugs and issues that were submitted prior to this initiative, to determine if they are already covered by our current gap analysis or if there are additional issues that need to be brought into the project. One issue has already been discovered and moved into the project, four others directly mapped to existing gap analysis issues and two others are expected to be closed by addressing existing issues already in the project that address the root cause of their issue.
+
+Thank you to all the community members who have participated. You are truly moving the needle in making React Native more accessible for everyone!
+
+## Closed Pull Requests 🎉
+
+- [Added talkback support for button accessibility: disabled prop #31001](https://github.com/facebook/react-native/pull/31001) - closed by [@huzaifaaak ](https://twitter.com/huzaifaaak)
+
+- [feat: set disabled accessibilityState when TouchableHighlight is disabled #31135](https://github.com/facebook/react-native/pull/31135) closed by [@natural_clar](https://twitter.com/natural_clar)
+
+- [[Android] Selected State does not annonce when TextInput Component selected #31144](https://github.com/facebook/react-native/pull/31144) closed by [fabriziobertoglio1987](http://fabriziobertoglio1987)
+
+- [Added talkback support for TouchableNativeFeedback accessibility: disabled prop #31224](https://github.com/facebook/react-native/pull/31224) closed by [@kyamashiro73](https://twitter.com/kyamashiro73)
+
+- [Accessibility/button test #31189](https://github.com/facebook/react-native/pull/31189) closed by [@huzaifaaak ](https://twitter.com/huzaifaaak)
+
+  - Adds a test for accessibilityState for button
+
+## Fixes
+
+- Button component (fixed by [#31001](https://github.com/facebook/react-native/pull/31001)):
+
+  - Now announces when it is disabled
+
+  - Disables click functionality for screen readers when the button is disabled
+
+  - Announces the selected state of the button
+
+- TouchableHighlight Component (fixed by [#31135](https://github.com/facebook/react-native/pull/31135)):
+
+  - Disables click functionality for screen readers when the component is disabled
+
+- TextInput Component Component (fixed by [#31144](https://github.com/facebook/react-native/pull/31144))
+
+  - Announces "selected" when the "selected" accessibilityState is set to true and the element is focused
+
+- TouchableNativeFeedback Component (fixed by [#31224](https://github.com/facebook/react-native/pull/31224))
+
+  - Disables click functionality for screen readers when the component is disabled
+
+## Other Progress
+
+To Do: 53
+
+In Progress Issues by the Community: 8
+
+In Progress Issues by React Native Team: 5
+
+Pull Request in Progress: 3
+
+Pull Request in Reviews: 4
```

Comment:
This section is looking a bit plan, what do you think about converting this part to table or list?

Reply:
Oh I did not know that. Thank you for the addition.

Reply:
I have switched it to a table, the list still looked a bit plain.

Reply:
Hmm, it looks a bit better, but maybe you can add few emojis to the statuses, something like:
<img width="275" alt="Screenshot 2021-04-08 143424" src="https://user-images.githubusercontent.com/719641/114027612-d8c12980-9877-11eb-9bd6-02cdb9baa091.png">

Reply:
Hmm I'm not sure if that would work since it would look like 

- 53 - ⏳ Issues To Do
- 8 - 🚧 In Progress Issues by the Community
- 5 - 🚧 In Progress Issues by React Native Team
- 3 - 🚧 Pull Request in Progress
- 4 - 💬 Pull Request in Reviews

Reply:
Sorry, the screenshot might be a bit confusing, I would like to see the emojis in table, here is my idea properly visualized:

<img width="584" alt="Screenshot 2021-04-08 145630" src="https://user-images.githubusercontent.com/719641/114030256-9fd68400-987a-11eb-98ee-88c7b3edf9b3.png">

But if you want to use same emojis for few entries, it might not be an visual improvement, so maybe the table should be left as-is right now. 🙂

Reply:
Ah ok let me add some different ones. I thought that the emojis you had listed were the only ones to use.

Reply:
Hmm it doesn't sound great when using a screen reader to hear the emoji description and then the title of the status even with different emojis for each status.

Reply:
That's a good point, so maybe just leave table as-is for now?

Reply:
Yes